src/Form/selectors-css.js

```diff
@@ -170,6 +170,12 @@ const birthdayYear = `
 [name=date_of_birth_year], [name=date-of-birth-year],
 [name^=birthdate_y], [name^=birthdate-y]`
 
+const username = [
+    `${GENERIC_TEXT_FIELD}[autocomplete^=user]`,
+    // fix for `aa.com`
+    `${GENERIC_TEXT_FIELD}[name="loginId"]`
```

Comment:
We don't need the `GENERIC_TEXT_FIELD` here, because the name is unambiguous and we are doing an exact match. We needed it above because we're doing a `^` matching. This new one is more straightforward.

Reply:
@GioSensation done